CMakeLists.txt

```diff
@@ -417,31 +411,31 @@ else()
             COMPONENT devel)
 endif()
 
-install(FILES license.md DESTINATION ${SFML_MISC_INSTALL_PREFIX})
-install(FILES readme.md DESTINATION ${SFML_MISC_INSTALL_PREFIX})
+install(FILES license.md DESTINATION ${CMAKE_INSTALL_INFODIR})
```

Comment:
`${CMAKE_INSTALL_INFODIR}` (aka `/usr/share/info`) is only for [Gnu TexInfo](https://www.gnu.org/software/texinfo/) files. Use `${CMAKE_INSTALL_DOCDIR}` instead.

Reply:
Thanks, Done